src/pages/QuizPage.scss

```
@@ -0,0 +1,124 @@
+@use "../styles/variables" as *;
+
+$planet-width-mobile: 4rem;
+$planet-width-tablet: 5rem;
+$planet-width-desktop: 8vw;
+
+$solar-system-height-mobile: 16vh;
+$solar-system-height-tablet: 10rem;
+$solar-system-height-desktop: 12vw;
+
+$solar-system-width-mobile: 80vw;
+$solar-system-width: 80vw;
+
+body {
+  margin: 0px;
+}
+
+.QuizPage {
+  max-width: 80vw;
+
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  justify-content: space-between;
+
+  &__solar-system {
+    background-color: black;
+    height: $solar-system-height-mobile;
+    width: calc(72vw + $planet-width-mobile);
+    margin: 0 auto;
+    position: relative;
+
+    &--hide {
+      display: none;
+    }
+  }
+}
+
+.ufo,
+.earth,
+.mars {
+  width: $planet-width-mobile;
+  height: $planet-width-mobile;
+  position: absolute;
+  top: calc(($solar-system-height-mobile - $planet-width-mobile) / 2);
+}
+
+.mars {
+  /* background-color: red; */
+  background: url(../assets/quiz-mars-cartoon.png), red;
+  border-radius: 50%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  left: 0;
+}
+.ufo {
+  background: url(../assets/quiz-ufo-cartoon.png);
+  /* border-radius: 50%; */
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  left: 0;
+}
+
+.earth {
+  /* background-color: green; */
+  background: url(../assets/quiz-earth-cartoon.png), blue;
```

Among others (I haven't commented everywhere), this will change to
```suggestion
  background: url(/quiz-earth-cartoon.png), blue;
```